apps/root/src/pages/transfer/components/transfer-form/index.tsx

```diff
@@ -44,6 +44,10 @@ import FormWalletSelector from '@common/components/form-wallet-selector';
 
 const StyledTransferForm = styled(BackgroundPaper)`
   position: relative;
+  ${({ theme: { space } }) => `
```

Comment:
Shouldn't this be integrated to the background paper? Seems like there are already rules into the spacing it should have